Content.Server/Verbs/InvokeVerbCommand.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Linq;
+using System.Net.Security;
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.Verbs;
+using Robust.Shared.Console;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Verbs
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public class InvokeVerbCommand : IConsoleCommand
+    {
+        public string Command => "invokeverb";
```

Comment:
Not strictly necessary for this command, but having an accompanying "list verbs" command would probably make this one much easier to use.

Reply:
Especially given that some verbs (like pickup) go through system provided functions that do their own can-interact checks. So even with force=true, you wouldn't necessarily know what verbs are valid.

Reply:
true, i'll add that as well